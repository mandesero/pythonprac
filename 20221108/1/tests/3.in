a = DivStr("ABCDАВАfdjbnsdjfbubJBSDBVYWBHfbhdsbvhidjnfvjnEFGHIJKLMNOPQRSTUVWXYZ")
print(* a // 2)
print(a % 3)
print(* a % 10 // 3)
print(a.lower() % 6)
print(* a[1:6] // 1)
print(DivStr())
print(a % 6 + DivStr() + a % 5)


