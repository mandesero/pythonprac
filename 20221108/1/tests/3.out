ABCDАВАfdjbnsdjfbubJBSDBVYWBHfbh dsbvhidjnfvjnEFGHIJKLMNOPQRSTUVWXYZ
Z
ABCDАВАfdjbnsdjfbubJ BSDBVYWBHfbhdsbvhidjnfv jnEFGHIJKLMNOPQRSTUVWXY
wxyz
BCDА

WXYZABCDАВАfdjbnsdjfbubJBSDBVYWBHfbhdsbvhidjnfvjnEFGHIJKLMNOPQRSTUVWXYZ
